src/components/site-header/SiteHeaderLogos.jsx

```diff
@@ -30,17 +29,20 @@ const SiteHeaderLogos = () => {
   }
 
   return (
-    <>
+    <Stack direction="horizontal" gap={3} className="mr-lg-3">
```

Comment:
Handles equal horizontal spacing between the main logo, the vertical divider, and the partner logo. The `mr-lg-3` only adds right margin between the logos and the menu nav items for the desktop view.

https://paragon-openedx.netlify.app/components/stack/